Assets/Scripts/Player/PlayerDash.cs

```
@@ -36,6 +36,15 @@ public class PlayerDash : MonoBehaviour
     [SerializeField]
     private bool _normalizeDiagonalDash = true;
 
+    [SerializeField]
+    private bool _ignoreCollisionsWithPlayers;
+
+    [SerializeField]
+    private bool _ignorePlayerCollisionsOnDash;
```

Can we throw an error if both of these values are true? From my understanding, you would only want one or the other to be true at any given time.